### DISCUSSION TITLE
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by freedom909** [prisma/prisma#21391] [Q&A]
### Bug description

Db: mongodb atlas
platform :nodejs
"@prisma/client": "^5.4.1",
when run "node bookings.js" command, the error message is:
```
PS C:\Users\takayama\Desktop\apollo\air0\subgraph-listings\datasources> node bookings.js
C:\Users\takayama\Desktop\apollo\air0\subgraph-listings\node_modules\.prisma\client\index.js:43
    throw new Error(
          ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
    at new PrismaClient (C:\Users\takayama\Desktop\apollo\air0\subgraph-listings\node_modules\.prisma\client\index.js:43:11)
    at new BookingsAPI (file:///C:/Users/takayama/Desktop/apollo/air0/subgraph-listings/datasources/bookings.js:12:17)
    at file:///C:/Users/takayama/Desktop/apollo/air0/subgraph-listings/datasources/bookings.js:136:1
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:228:24)
    at async loadESM (node:internal/process/esm_loader:40:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)
```

Node.js v20.5.0

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @freedom909 👋 

Thank you for raising this question.

According to the error message:

> Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

You will need to run the command `npx prisma generate`.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by freedom909** [comment]
thank you very much

Sent from Outlook<http://aka.ms/weboutlook>
________________________________

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
